README.md

```diff
@@ -54,8 +55,7 @@ Read this [wiki page](https://github.com/RocketChat/EmbeddedChat/wiki/Roots-of-E
 | roomId           | the public room's id that you want to subscribe to                                                                                          |
 | channelName      | the fallback channel name to be present on the chat header                                                                                  |
 | anonymousMode    | if the user can see the chat without logging in                                                                                             |
-| showAvatar       | show the user's avatar in the chat component, defaults to false                                                      |
-
+| showAvatar       | show the user's avatar in the chat component, defaults to false                                                                             |
```

Comment:
can you please add a line here on the description part?

Reply:
resolved in [b3f1282](https://github.com/RocketChat/EmbeddedChat/pull/115/commits/b3f12828265f0806f03ef0edf2e5b96cd06c0c31)